var/spack/repos/builtin/packages/intel-oneapi-compilers/package.py

```diff
@@ -329,6 +331,46 @@ class IntelOneapiCompilers(IntelOneApiPackage, CompilerPackage):
         r"(?:(?:oneAPI DPC\+\+(?:\/C\+\+)? Compiler)|(?:\(IFORT\))|(?:\(IFX\))) (\S+)"
     )
 
+    debug_flags = ["-debug", "-g", "-g0", "-g1", "-g2", "-g3"]
+    opt_flags = ["-O", "-O0", "-O1", "-O2", "-O3", "-Ofast", "-Os"]
+
+    openmp_flag = "-fiopenmp"
+
+    compiler_wrapper_link_paths = {
+        "c": os.path.join("oneapi", "icx"),
+        "cxx": os.path.join("oneapi", "icpx"),
+        "fortran": os.path.join("oneapi", "ifx"),
+    }
+
+    required_libs = [
```

Comment:
I still think this name (used only in implicit RPATH detection) is confusing.

Reply:
Let's change this to `implicit_rpath_libs` and document it.

Reply:
See c27eb174630e2b8a468da91abe47797a679873d9